.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 8,
```

Comment:
If/when we change this again:  I believe the community preference is to use the year (2017 vs 8) ... it's certainly the preference of TC39 to version by year (-: